README.md

```diff
@@ -1,2 +1,59 @@
-# portfolio
-This is my portfolio project
+# hello-microverse-project-
+
+> This project was made using flexbox and it was based on Template 3. The aim of the project is to build a personal portfolio site
+
+
+## Built With
+
+- HTML
+- CSS
+- linter
+
+## Live Demo (if available)
+
+No
+
```

Comment:
- [ ] [Optional] Kindly consider adding a working live preview link.🙂
**Note:** _Instead of using 'No', you can be more descriptive. For example, 'Not Available'_